src/parsers/init.js

```diff
@@ -0,0 +1,28 @@
+class BaseParser {
+    static property = {};
+
+    constructor(content) {
+        this.content = content;
+    }
+
+    static parse() {
```

Comment:
Should be function of two parameters:

- content
- property_name

https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/src/xmlparser/XMLParser.js